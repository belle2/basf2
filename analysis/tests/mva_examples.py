#!/usr/bin/env python3
# -- analysis/tests/mva_examples.py

##########################################################################
# basf2 (Belle II Analysis Software Framework)                           #
# Author: The Belle II Collaboration                                     #
#                                                                        #
# See git log for contributors and copyright holders.                    #
# This file is licensed under LGPL-3.0, see LICENSE.md.                  #
##########################################################################

import b2test_utils as b2tu
import b2test_utils_analysis as b2tua

#: extend the examples test class to actually run over the mva examples:


class ExamplesTest(b2tua.ExamplesTest):
    def test_mva_examples(self):
        """
        Test supported mva examples.
        """
        # list of examples that cannot be tested
        broken_mva_egs = ['B2A712-DeepContinuumSuppression_MVATrain.py',  # training takes too long and cannot be shortened
                          'B2A713-DeepContinuumSuppression_MVAExpert.py',  # requires weight file created in previous example
                          'B2A714-DeepContinuumSuppression_MVAModel.py'  # MVA model description and not standalone example script
                          ]

        self._test_examples_dir('analysis/examples/mva/', broken_mva_egs)


if __name__ == '__main__':
    with b2tu.clean_working_directory():
        b2tua.unittest.main()
