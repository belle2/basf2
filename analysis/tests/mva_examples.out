[INFO] Steering file: analysis/tests/mva_examples.py
TFile: B2A711-DeepContinuumSuppression_Input.root
TTree: tree
TBranch: __experiment__, 0
TBranch: __run__, 0
TBranch: __event__, 488480075
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 1
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: isContinuumEvent, True
TBranch: isNotContinuumEvent, False
TBranch: isSignal, 0
TBranch: M, 5.114
TBranch: p, 2.032
TBranch: Mbc, 5.251
TBranch: DeltaZ, nan
TBranch: deltaE, -0.1361
TBranch: daughter__bo0__cm__spM__bc, 0.4984
TBranch: daughter__bo0__cm__spp__bc, 3.047
TBranch: daughter__bo1__cm__spM__bc, 0.1297
TBranch: daughter__bo1__cm__spp__bc, 2.412
TBranch: clusterNHits_Croe0, 8.008
TBranch: clusterTiming_Croe0, 5.371
TBranch: clusterE9E25_Croe0, 0.9365
TBranch: clusterReg_Croe0, 2
TBranch: isInRestOfEvent_Croe0, 1
TBranch: thrustsigp_Croe0, 0.2479
TBranch: thrustsigphi_Croe0, -2.519
TBranch: thrustsigcosTheta_Croe0, 0.9555
TBranch: thrustsigpErr_Croe0, 0.009894
TBranch: thrustsigphiErr_Croe0, 0.02456
TBranch: thrustsigcosThetaErr_Croe0, 0.002089
TBranch: clusterNHits_Csig0, 14.06
TBranch: clusterTiming_Csig0, 0
TBranch: clusterE9E25_Csig0, 0.9814
TBranch: clusterReg_Csig0, 2
TBranch: isInRestOfEvent_Csig0, 0
TBranch: thrustsigp_Csig0, 2.432
TBranch: thrustsigphi_Csig0, 2.466
TBranch: thrustsigcosTheta_Csig0, -0.9907
TBranch: thrustsigpErr_Csig0, 0.04564
TBranch: thrustsigphiErr_Csig0, 0.02018
TBranch: thrustsigcosThetaErr_Csig0, 0.0003739
TBranch: clusterNHits_Croe1, 6.055
TBranch: clusterTiming_Croe1, 8.301
TBranch: clusterE9E25_Croe1, 1
TBranch: clusterReg_Croe1, 2
TBranch: isInRestOfEvent_Croe1, 1
TBranch: thrustsigp_Croe1, 0.1837
TBranch: thrustsigphi_Croe1, -1.057
TBranch: thrustsigcosTheta_Croe1, 0.1364
TBranch: thrustsigpErr_Croe1, 0.0107
TBranch: thrustsigphiErr_Croe1, 0.007177
TBranch: thrustsigcosThetaErr_Croe1, 0.007333
TBranch: clusterNHits_Csig1, 10.94
TBranch: clusterTiming_Csig1, 21.97
TBranch: clusterE9E25_Csig1, 0.8906
TBranch: clusterReg_Csig1, 2
TBranch: isInRestOfEvent_Csig1, 0
TBranch: thrustsigp_Csig1, 0.1512
TBranch: thrustsigphi_Csig1, -0.183
TBranch: thrustsigcosTheta_Csig1, -0.9965
TBranch: thrustsigpErr_Csig1, 0.009945
TBranch: thrustsigphiErr_Csig1, 0.1071
TBranch: thrustsigcosThetaErr_Csig1, 0.0006113
TBranch: clusterNHits_Croe2, 7.031
TBranch: clusterTiming_Croe2, -470.7
TBranch: clusterE9E25_Croe2, 0.916
TBranch: clusterReg_Croe2, 2
TBranch: isInRestOfEvent_Croe2, 1
TBranch: thrustsigp_Croe2, 0.05387
TBranch: thrustsigphi_Croe2, 2.944
TBranch: thrustsigcosTheta_Croe2, -0.4998
TBranch: thrustsigpErr_Croe2, 0.005417
TBranch: thrustsigphiErr_Croe2, 0.01073
TBranch: thrustsigcosThetaErr_Croe2, 0.01105
TBranch: clusterNHits_Csig2, nan
TBranch: clusterTiming_Csig2, nan
TBranch: clusterE9E25_Csig2, nan
TBranch: clusterReg_Csig2, nan
TBranch: isInRestOfEvent_Csig2, nan
TBranch: thrustsigp_Csig2, nan
TBranch: thrustsigphi_Csig2, nan
TBranch: thrustsigcosTheta_Csig2, nan
TBranch: thrustsigpErr_Csig2, nan
TBranch: thrustsigphiErr_Csig2, nan
TBranch: thrustsigcosThetaErr_Csig2, nan
TBranch: clusterNHits_Croe3, 6.055
TBranch: clusterTiming_Croe3, -14.65
TBranch: clusterE9E25_Croe3, 0.9023
TBranch: clusterReg_Croe3, 2
TBranch: isInRestOfEvent_Croe3, 1
TBranch: thrustsigp_Croe3, 0.05332
TBranch: thrustsigphi_Croe3, -2.532
TBranch: thrustsigcosTheta_Croe3, -0.4358
TBranch: thrustsigpErr_Croe3, 0.005492
TBranch: thrustsigphiErr_Croe3, 0.01339
TBranch: thrustsigcosThetaErr_Croe3, 0.01069
TBranch: clusterNHits_Csig3, nan
TBranch: clusterTiming_Csig3, nan
TBranch: clusterE9E25_Csig3, nan
TBranch: clusterReg_Csig3, nan
TBranch: isInRestOfEvent_Csig3, nan
TBranch: thrustsigp_Csig3, nan
TBranch: thrustsigphi_Csig3, nan
TBranch: thrustsigcosTheta_Csig3, nan
TBranch: thrustsigpErr_Csig3, nan
TBranch: thrustsigphiErr_Csig3, nan
TBranch: thrustsigcosThetaErr_Csig3, nan
TBranch: clusterNHits_Croe4, nan
TBranch: clusterTiming_Croe4, nan
TBranch: clusterE9E25_Croe4, nan
TBranch: clusterReg_Croe4, nan
TBranch: isInRestOfEvent_Croe4, nan
TBranch: thrustsigp_Croe4, nan
TBranch: thrustsigphi_Croe4, nan
TBranch: thrustsigcosTheta_Croe4, nan
TBranch: thrustsigpErr_Croe4, nan
TBranch: thrustsigphiErr_Croe4, nan
TBranch: thrustsigcosThetaErr_Croe4, nan
TBranch: clusterNHits_Csig4, nan
TBranch: clusterTiming_Csig4, nan
TBranch: clusterE9E25_Csig4, nan
TBranch: clusterReg_Csig4, nan
TBranch: isInRestOfEvent_Csig4, nan
TBranch: thrustsigp_Csig4, nan
TBranch: thrustsigphi_Csig4, nan
TBranch: thrustsigcosTheta_Csig4, nan
TBranch: thrustsigpErr_Csig4, nan
TBranch: thrustsigphiErr_Csig4, nan
TBranch: thrustsigcosThetaErr_Csig4, nan
TBranch: clusterNHits_Croe5, nan
TBranch: clusterTiming_Croe5, nan
TBranch: clusterE9E25_Croe5, nan
TBranch: clusterReg_Croe5, nan
TBranch: isInRestOfEvent_Croe5, nan
TBranch: thrustsigp_Croe5, nan
TBranch: thrustsigphi_Croe5, nan
TBranch: thrustsigcosTheta_Croe5, nan
TBranch: thrustsigpErr_Croe5, nan
TBranch: thrustsigphiErr_Croe5, nan
TBranch: thrustsigcosThetaErr_Croe5, nan
TBranch: clusterNHits_Csig5, nan
TBranch: clusterTiming_Csig5, nan
TBranch: clusterE9E25_Csig5, nan
TBranch: clusterReg_Csig5, nan
TBranch: isInRestOfEvent_Csig5, nan
TBranch: thrustsigp_Csig5, nan
TBranch: thrustsigphi_Csig5, nan
TBranch: thrustsigcosTheta_Csig5, nan
TBranch: thrustsigpErr_Csig5, nan
TBranch: thrustsigphiErr_Csig5, nan
TBranch: thrustsigcosThetaErr_Csig5, nan
TBranch: clusterNHits_Croe6, nan
TBranch: clusterTiming_Croe6, nan
TBranch: clusterE9E25_Croe6, nan
TBranch: clusterReg_Croe6, nan
TBranch: isInRestOfEvent_Croe6, nan
TBranch: thrustsigp_Croe6, nan
TBranch: thrustsigphi_Croe6, nan
TBranch: thrustsigcosTheta_Croe6, nan
TBranch: thrustsigpErr_Croe6, nan
TBranch: thrustsigphiErr_Croe6, nan
TBranch: thrustsigcosThetaErr_Croe6, nan
TBranch: clusterNHits_Csig6, nan
TBranch: clusterTiming_Csig6, nan
TBranch: clusterE9E25_Csig6, nan
TBranch: clusterReg_Csig6, nan
TBranch: isInRestOfEvent_Csig6, nan
TBranch: thrustsigp_Csig6, nan
TBranch: thrustsigphi_Csig6, nan
TBranch: thrustsigcosTheta_Csig6, nan
TBranch: thrustsigpErr_Csig6, nan
TBranch: thrustsigphiErr_Csig6, nan
TBranch: thrustsigcosThetaErr_Csig6, nan
TBranch: clusterNHits_Croe7, nan
TBranch: clusterTiming_Croe7, nan
TBranch: clusterE9E25_Croe7, nan
TBranch: clusterReg_Croe7, nan
TBranch: isInRestOfEvent_Croe7, nan
TBranch: thrustsigp_Croe7, nan
TBranch: thrustsigphi_Croe7, nan
TBranch: thrustsigcosTheta_Croe7, nan
TBranch: thrustsigpErr_Croe7, nan
TBranch: thrustsigphiErr_Croe7, nan
TBranch: thrustsigcosThetaErr_Croe7, nan
TBranch: clusterNHits_Csig7, nan
TBranch: clusterTiming_Csig7, nan
TBranch: clusterE9E25_Csig7, nan
TBranch: clusterReg_Csig7, nan
TBranch: isInRestOfEvent_Csig7, nan
TBranch: thrustsigp_Csig7, nan
TBranch: thrustsigphi_Csig7, nan
TBranch: thrustsigcosTheta_Csig7, nan
TBranch: thrustsigpErr_Csig7, nan
TBranch: thrustsigphiErr_Csig7, nan
TBranch: thrustsigcosThetaErr_Csig7, nan
TBranch: clusterNHits_Croe8, nan
TBranch: clusterTiming_Croe8, nan
TBranch: clusterE9E25_Croe8, nan
TBranch: clusterReg_Croe8, nan
TBranch: isInRestOfEvent_Croe8, nan
TBranch: thrustsigp_Croe8, nan
TBranch: thrustsigphi_Croe8, nan
TBranch: thrustsigcosTheta_Croe8, nan
TBranch: thrustsigpErr_Croe8, nan
TBranch: thrustsigphiErr_Croe8, nan
TBranch: thrustsigcosThetaErr_Croe8, nan
TBranch: clusterNHits_Csig8, nan
TBranch: clusterTiming_Csig8, nan
TBranch: clusterE9E25_Csig8, nan
TBranch: clusterReg_Csig8, nan
TBranch: isInRestOfEvent_Csig8, nan
TBranch: thrustsigp_Csig8, nan
TBranch: thrustsigphi_Csig8, nan
TBranch: thrustsigcosTheta_Csig8, nan
TBranch: thrustsigpErr_Csig8, nan
TBranch: thrustsigphiErr_Csig8, nan
TBranch: thrustsigcosThetaErr_Csig8, nan
TBranch: clusterNHits_Croe9, nan
TBranch: clusterTiming_Croe9, nan
TBranch: clusterE9E25_Croe9, nan
TBranch: clusterReg_Croe9, nan
TBranch: isInRestOfEvent_Croe9, nan
TBranch: thrustsigp_Croe9, nan
TBranch: thrustsigphi_Croe9, nan
TBranch: thrustsigcosTheta_Croe9, nan
TBranch: thrustsigpErr_Croe9, nan
TBranch: thrustsigphiErr_Croe9, nan
TBranch: thrustsigcosThetaErr_Croe9, nan
TBranch: clusterNHits_Csig9, nan
TBranch: clusterTiming_Csig9, nan
TBranch: clusterE9E25_Csig9, nan
TBranch: clusterReg_Csig9, nan
TBranch: isInRestOfEvent_Csig9, nan
TBranch: thrustsigp_Csig9, nan
TBranch: thrustsigphi_Csig9, nan
TBranch: thrustsigcosTheta_Csig9, nan
TBranch: thrustsigpErr_Csig9, nan
TBranch: thrustsigphiErr_Csig9, nan
TBranch: thrustsigcosThetaErr_Csig9, nan
TBranch: kaonID_TProe0, 5.918e-06
TBranch: electronID_TProe0, 2.231e-06
TBranch: muonID_TProe0, 7.279e-08
TBranch: protonID_TProe0, 0.0004107
TBranch: pValue_TProe0, 0.1214
TBranch: nCDCHits_TProe0, 12
TBranch: isInRestOfEvent_TProe0, 1
TBranch: charge_TProe0, 1
TBranch: thrustsigp_TProe0, 2.195
TBranch: thrustsigphi_TProe0, 2.829
TBranch: thrustsigcosTheta_TProe0, 0.02193
TBranch: thrustsigpErr_TProe0, 0.6652
TBranch: thrustsigphiErr_TProe0, 0.05867
TBranch: thrustsigcosThetaErr_TProe0, 0.05847
TBranch: thrustsigdistance_TProe0, 283.9
TBranch: thrustsigdphi_TProe0, -0.2894
TBranch: thrustsigdcosTheta_TProe0, -0.351
TBranch: kaonID_TPsig0, 0.9988
TBranch: electronID_TPsig0, 5.987e-36
TBranch: muonID_TPsig0, 3.381e-11
TBranch: protonID_TPsig0, 9.893e-07
TBranch: pValue_TPsig0, 0.06076
TBranch: nCDCHits_TPsig0, 55
TBranch: isInRestOfEvent_TPsig0, 0
TBranch: charge_TPsig0, -1
TBranch: thrustsigp_TPsig0, 2.259
TBranch: thrustsigphi_TPsig0, 2.217
TBranch: thrustsigcosTheta_TPsig0, 0.9906
TBranch: thrustsigpErr_TPsig0, 0.005505
TBranch: thrustsigphiErr_TPsig0, 0.004659
TBranch: thrustsigcosThetaErr_TPsig0, 8.697e-05
TBranch: thrustsigdistance_TPsig0, 0.03692
TBranch: thrustsigdphi_TPsig0, -0.7166
TBranch: thrustsigdcosTheta_TPsig0, -0.2866
TBranch: kaonID_TMroe0, 5.918e-06
TBranch: electronID_TMroe0, 2.231e-06
TBranch: muonID_TMroe0, 7.279e-08
TBranch: protonID_TMroe0, 0.0004107
TBranch: pValue_TMroe0, 0.1214
TBranch: nCDCHits_TMroe0, 12
TBranch: isInRestOfEvent_TMroe0, 1
TBranch: charge_TMroe0, 1
TBranch: thrustsigp_TMroe0, 2.195
TBranch: thrustsigphi_TMroe0, 2.829
TBranch: thrustsigcosTheta_TMroe0, 0.02193
TBranch: thrustsigpErr_TMroe0, 0.6652
TBranch: thrustsigphiErr_TMroe0, 0.05867
TBranch: thrustsigcosThetaErr_TMroe0, 0.05847
TBranch: thrustsigdistance_TMroe0, 283.9
TBranch: thrustsigdphi_TMroe0, -0.2894
TBranch: thrustsigdcosTheta_TMroe0, -0.351
TBranch: kaonID_TMsig0, 0.9988
TBranch: electronID_TMsig0, 5.987e-36
TBranch: muonID_TMsig0, 3.381e-11
TBranch: protonID_TMsig0, 9.893e-07
TBranch: pValue_TMsig0, 0.06076
TBranch: nCDCHits_TMsig0, 55
TBranch: isInRestOfEvent_TMsig0, 0
TBranch: charge_TMsig0, -1
TBranch: thrustsigp_TMsig0, 2.259
TBranch: thrustsigphi_TMsig0, 2.217
TBranch: thrustsigcosTheta_TMsig0, 0.9906
TBranch: thrustsigpErr_TMsig0, 0.005505
TBranch: thrustsigphiErr_TMsig0, 0.004659
TBranch: thrustsigcosThetaErr_TMsig0, 8.697e-05
TBranch: thrustsigdistance_TMsig0, 0.03692
TBranch: thrustsigdphi_TMsig0, -0.7166
TBranch: thrustsigdcosTheta_TMsig0, -0.2866
TBranch: kaonID_TProe1, 1
TBranch: electronID_TProe1, 4.728e-45
TBranch: muonID_TProe1, 1.18e-07
TBranch: protonID_TProe1, 2.472e-19
TBranch: pValue_TProe1, 0.2636
TBranch: nCDCHits_TProe1, 52
TBranch: isInRestOfEvent_TProe1, 1
TBranch: charge_TProe1, 1
TBranch: thrustsigp_TProe1, 1.333
TBranch: thrustsigphi_TProe1, 2.901
TBranch: thrustsigcosTheta_TProe1, -0.9822
TBranch: thrustsigpErr_TProe1, 0.002446
TBranch: thrustsigphiErr_TProe1, 0.004726
TBranch: thrustsigcosThetaErr_TProe1, 0.000177
TBranch: thrustsigdistance_TProe1, 0.00807
TBranch: thrustsigdphi_TProe1, -0.7834
TBranch: thrustsigdcosTheta_TProe1, -0.3105
TBranch: kaonID_TPsig1, 1.824e-37
TBranch: electronID_TPsig1, 1
TBranch: muonID_TPsig1, 4.275e-12
TBranch: protonID_TPsig1, 3.36e-28
TBranch: pValue_TPsig1, 0.02875
TBranch: nCDCHits_TPsig1, 91
TBranch: isInRestOfEvent_TPsig1, 0
TBranch: charge_TPsig1, 1
TBranch: thrustsigp_TPsig1, 0.2727
TBranch: thrustsigphi_TPsig1, -2.529
TBranch: thrustsigcosTheta_TPsig1, 0.9637
TBranch: thrustsigpErr_TPsig1, 0.0006652
TBranch: thrustsigphiErr_TPsig1, 0.0211
TBranch: thrustsigcosThetaErr_TPsig1, 0.001503
TBranch: thrustsigdistance_TPsig1, 0.0482
TBranch: thrustsigdphi_TPsig1, -0.1037
TBranch: thrustsigdcosTheta_TPsig1, -0.3088
TBranch: kaonID_TMroe1, 1
TBranch: electronID_TMroe1, 4.728e-45
TBranch: muonID_TMroe1, 1.18e-07
TBranch: protonID_TMroe1, 2.472e-19
TBranch: pValue_TMroe1, 0.2636
TBranch: nCDCHits_TMroe1, 52
TBranch: isInRestOfEvent_TMroe1, 1
TBranch: charge_TMroe1, 1
TBranch: thrustsigp_TMroe1, 1.333
TBranch: thrustsigphi_TMroe1, 2.901
TBranch: thrustsigcosTheta_TMroe1, -0.9822
TBranch: thrustsigpErr_TMroe1, 0.002446
TBranch: thrustsigphiErr_TMroe1, 0.004726
TBranch: thrustsigcosThetaErr_TMroe1, 0.000177
TBranch: thrustsigdistance_TMroe1, 0.00807
TBranch: thrustsigdphi_TMroe1, -0.7834
TBranch: thrustsigdcosTheta_TMroe1, -0.3105
TBranch: kaonID_TMsig1, 1.824e-37
TBranch: electronID_TMsig1, 1
TBranch: muonID_TMsig1, 4.275e-12
TBranch: protonID_TMsig1, 3.36e-28
TBranch: pValue_TMsig1, 0.02875
TBranch: nCDCHits_TMsig1, 91
TBranch: isInRestOfEvent_TMsig1, 0
TBranch: charge_TMsig1, 1
TBranch: thrustsigp_TMsig1, 0.2727
TBranch: thrustsigphi_TMsig1, -2.529
TBranch: thrustsigcosTheta_TMsig1, 0.9637
TBranch: thrustsigpErr_TMsig1, 0.0006652
TBranch: thrustsigphiErr_TMsig1, 0.0211
TBranch: thrustsigcosThetaErr_TMsig1, 0.001503
TBranch: thrustsigdistance_TMsig1, 0.0482
TBranch: thrustsigdphi_TMsig1, -0.1037
TBranch: thrustsigdcosTheta_TMsig1, -0.3088
TBranch: kaonID_TProe2, 7.259e-09
TBranch: electronID_TProe2, 1.396e-16
TBranch: muonID_TProe2, 8.107e-227
TBranch: protonID_TProe2, 3.505e-51
TBranch: pValue_TProe2, 0.1149
TBranch: nCDCHits_TProe2, 57
TBranch: isInRestOfEvent_TProe2, 1
TBranch: charge_TProe2, 1
TBranch: thrustsigp_TProe2, 0.7903
TBranch: thrustsigphi_TProe2, -0.7196
TBranch: thrustsigcosTheta_TProe2, 0.8031
TBranch: thrustsigpErr_TProe2, 0.001366
TBranch: thrustsigphiErr_TProe2, 0.002382
TBranch: thrustsigcosThetaErr_TProe2, 0.0008528
TBranch: thrustsigdistance_TProe2, 0.03842
TBranch: thrustsigdphi_TProe2, 2.544
TBranch: thrustsigdcosTheta_TProe2, 0.4018
TBranch: kaonID_TPsig2, nan
TBranch: electronID_TPsig2, nan
TBranch: muonID_TPsig2, nan
TBranch: protonID_TPsig2, nan
TBranch: pValue_TPsig2, nan
TBranch: nCDCHits_TPsig2, nan
TBranch: isInRestOfEvent_TPsig2, nan
TBranch: charge_TPsig2, nan
TBranch: thrustsigp_TPsig2, nan
TBranch: thrustsigphi_TPsig2, nan
TBranch: thrustsigcosTheta_TPsig2, nan
TBranch: thrustsigpErr_TPsig2, nan
TBranch: thrustsigphiErr_TPsig2, nan
TBranch: thrustsigcosThetaErr_TPsig2, nan
TBranch: thrustsigdistance_TPsig2, nan
TBranch: thrustsigdphi_TPsig2, nan
TBranch: thrustsigdcosTheta_TPsig2, nan
TBranch: kaonID_TMroe2, 7.259e-09
TBranch: electronID_TMroe2, 1.396e-16
TBranch: muonID_TMroe2, 8.107e-227
TBranch: protonID_TMroe2, 3.505e-51
TBranch: pValue_TMroe2, 0.1149
TBranch: nCDCHits_TMroe2, 57
TBranch: isInRestOfEvent_TMroe2, 1
TBranch: charge_TMroe2, 1
TBranch: thrustsigp_TMroe2, 0.7903
TBranch: thrustsigphi_TMroe2, -0.7196
TBranch: thrustsigcosTheta_TMroe2, 0.8031
TBranch: thrustsigpErr_TMroe2, 0.001366
TBranch: thrustsigphiErr_TMroe2, 0.002382
TBranch: thrustsigcosThetaErr_TMroe2, 0.0008528
TBranch: thrustsigdistance_TMroe2, 0.03842
TBranch: thrustsigdphi_TMroe2, 2.544
TBranch: thrustsigdcosTheta_TMroe2, 0.4018
TBranch: kaonID_TMsig2, nan
TBranch: electronID_TMsig2, nan
TBranch: muonID_TMsig2, nan
TBranch: protonID_TMsig2, nan
TBranch: pValue_TMsig2, nan
TBranch: nCDCHits_TMsig2, nan
TBranch: isInRestOfEvent_TMsig2, nan
TBranch: charge_TMsig2, nan
TBranch: thrustsigp_TMsig2, nan
TBranch: thrustsigphi_TMsig2, nan
TBranch: thrustsigcosTheta_TMsig2, nan
TBranch: thrustsigpErr_TMsig2, nan
TBranch: thrustsigphiErr_TMsig2, nan
TBranch: thrustsigcosThetaErr_TMsig2, nan
TBranch: thrustsigdistance_TMsig2, nan
TBranch: thrustsigdphi_TMsig2, nan
TBranch: thrustsigdcosTheta_TMsig2, nan
TBranch: kaonID_TProe3, 6.133e-08
TBranch: electronID_TProe3, 9.184e-10
TBranch: muonID_TProe3, 1.627e-10
TBranch: protonID_TProe3, 0.9942
TBranch: pValue_TProe3, 2.031e-09
TBranch: nCDCHits_TProe3, 0
TBranch: isInRestOfEvent_TProe3, 1
TBranch: charge_TProe3, -1
TBranch: thrustsigp_TProe3, 0.7537
TBranch: thrustsigphi_TProe3, 2.271
TBranch: thrustsigcosTheta_TProe3, -0.3975
TBranch: thrustsigpErr_TProe3, 0.0333
TBranch: thrustsigphiErr_TProe3, 0.002064
TBranch: thrustsigcosThetaErr_TProe3, 0.001654
TBranch: thrustsigdistance_TProe3, 0.005999
TBranch: thrustsigdphi_TProe3, 0.2795
TBranch: thrustsigdcosTheta_TProe3, 0.1828
TBranch: kaonID_TPsig3, nan
TBranch: electronID_TPsig3, nan
TBranch: muonID_TPsig3, nan
TBranch: protonID_TPsig3, nan
TBranch: pValue_TPsig3, nan
TBranch: nCDCHits_TPsig3, nan
TBranch: isInRestOfEvent_TPsig3, nan
TBranch: charge_TPsig3, nan
TBranch: thrustsigp_TPsig3, nan
TBranch: thrustsigphi_TPsig3, nan
TBranch: thrustsigcosTheta_TPsig3, nan
TBranch: thrustsigpErr_TPsig3, nan
TBranch: thrustsigphiErr_TPsig3, nan
TBranch: thrustsigcosThetaErr_TPsig3, nan
TBranch: thrustsigdistance_TPsig3, nan
TBranch: thrustsigdphi_TPsig3, nan
TBranch: thrustsigdcosTheta_TPsig3, nan
TBranch: kaonID_TMroe3, 6.133e-08
TBranch: electronID_TMroe3, 9.184e-10
TBranch: muonID_TMroe3, 1.627e-10
TBranch: protonID_TMroe3, 0.9942
TBranch: pValue_TMroe3, 2.031e-09
TBranch: nCDCHits_TMroe3, 0
TBranch: isInRestOfEvent_TMroe3, 1
TBranch: charge_TMroe3, -1
TBranch: thrustsigp_TMroe3, 0.7537
TBranch: thrustsigphi_TMroe3, 2.271
TBranch: thrustsigcosTheta_TMroe3, -0.3975
TBranch: thrustsigpErr_TMroe3, 0.0333
TBranch: thrustsigphiErr_TMroe3, 0.002064
TBranch: thrustsigcosThetaErr_TMroe3, 0.001654
TBranch: thrustsigdistance_TMroe3, 0.005999
TBranch: thrustsigdphi_TMroe3, 0.2795
TBranch: thrustsigdcosTheta_TMroe3, 0.1828
TBranch: kaonID_TMsig3, nan
TBranch: electronID_TMsig3, nan
TBranch: muonID_TMsig3, nan
TBranch: protonID_TMsig3, nan
TBranch: pValue_TMsig3, nan
TBranch: nCDCHits_TMsig3, nan
TBranch: isInRestOfEvent_TMsig3, nan
TBranch: charge_TMsig3, nan
TBranch: thrustsigp_TMsig3, nan
TBranch: thrustsigphi_TMsig3, nan
TBranch: thrustsigcosTheta_TMsig3, nan
TBranch: thrustsigpErr_TMsig3, nan
TBranch: thrustsigphiErr_TMsig3, nan
TBranch: thrustsigcosThetaErr_TMsig3, nan
TBranch: thrustsigdistance_TMsig3, nan
TBranch: thrustsigdphi_TMsig3, nan
TBranch: thrustsigdcosTheta_TMsig3, nan
TBranch: kaonID_TProe4, 5.352e-15
TBranch: electronID_TProe4, 1.276e-12
TBranch: muonID_TProe4, 0.0008952
TBranch: protonID_TProe4, 9.783e-50
TBranch: pValue_TProe4, 0.7958
TBranch: nCDCHits_TProe4, 36
TBranch: isInRestOfEvent_TProe4, 1
TBranch: charge_TProe4, -1
TBranch: thrustsigp_TProe4, 0.5912
TBranch: thrustsigphi_TProe4, 2.297
TBranch: thrustsigcosTheta_TProe4, -0.9193
TBranch: thrustsigpErr_TProe4, 0.001889
TBranch: thrustsigphiErr_TProe4, 0.01008
TBranch: thrustsigcosThetaErr_TProe4, 0.002216
TBranch: thrustsigdistance_TProe4, 11.06
TBranch: thrustsigdphi_TProe4, 2.373
TBranch: thrustsigdcosTheta_TProe4, 0.4064
TBranch: kaonID_TPsig4, nan
TBranch: electronID_TPsig4, nan
TBranch: muonID_TPsig4, nan
TBranch: protonID_TPsig4, nan
TBranch: pValue_TPsig4, nan
TBranch: nCDCHits_TPsig4, nan
TBranch: isInRestOfEvent_TPsig4, nan
TBranch: charge_TPsig4, nan
TBranch: thrustsigp_TPsig4, nan
TBranch: thrustsigphi_TPsig4, nan
TBranch: thrustsigcosTheta_TPsig4, nan
TBranch: thrustsigpErr_TPsig4, nan
TBranch: thrustsigphiErr_TPsig4, nan
TBranch: thrustsigcosThetaErr_TPsig4, nan
TBranch: thrustsigdistance_TPsig4, nan
TBranch: thrustsigdphi_TPsig4, nan
TBranch: thrustsigdcosTheta_TPsig4, nan
TBranch: kaonID_TMroe4, 5.352e-15
TBranch: electronID_TMroe4, 1.276e-12
TBranch: muonID_TMroe4, 0.0008952
TBranch: protonID_TMroe4, 9.783e-50
TBranch: pValue_TMroe4, 0.7958
TBranch: nCDCHits_TMroe4, 36
TBranch: isInRestOfEvent_TMroe4, 1
TBranch: charge_TMroe4, -1
TBranch: thrustsigp_TMroe4, 0.5912
TBranch: thrustsigphi_TMroe4, 2.297
TBranch: thrustsigcosTheta_TMroe4, -0.9193
TBranch: thrustsigpErr_TMroe4, 0.001889
TBranch: thrustsigphiErr_TMroe4, 0.01008
TBranch: thrustsigcosThetaErr_TMroe4, 0.002216
TBranch: thrustsigdistance_TMroe4, 11.06
TBranch: thrustsigdphi_TMroe4, 2.373
TBranch: thrustsigdcosTheta_TMroe4, 0.4064
TBranch: kaonID_TMsig4, nan
TBranch: electronID_TMsig4, nan
TBranch: muonID_TMsig4, nan
TBranch: protonID_TMsig4, nan
TBranch: pValue_TMsig4, nan
TBranch: nCDCHits_TMsig4, nan
TBranch: isInRestOfEvent_TMsig4, nan
TBranch: charge_TMsig4, nan
TBranch: thrustsigp_TMsig4, nan
TBranch: thrustsigphi_TMsig4, nan
TBranch: thrustsigcosTheta_TMsig4, nan
TBranch: thrustsigpErr_TMsig4, nan
TBranch: thrustsigphiErr_TMsig4, nan
TBranch: thrustsigcosThetaErr_TMsig4, nan
TBranch: thrustsigdistance_TMsig4, nan
TBranch: thrustsigdphi_TMsig4, nan
TBranch: thrustsigdcosTheta_TMsig4, nan
TBranch: R2, 0.2896
TBranch: thrustBm, 0.9894
TBranch: thrustOm, 0.7283
TBranch: cosTBTO, 0.5776
TBranch: cosTBz, 0.3408
TBranch: KSFWVariables__boet__bc, 9.376
TBranch: KSFWVariables__bomm2__bc, -9.255
TBranch: KSFWVariables__bohso00__bc, 1.037
TBranch: KSFWVariables__bohso02__bc, 0.1332
TBranch: KSFWVariables__bohso04__bc, 0.3572
TBranch: KSFWVariables__bohso10__bc, 0.1488
TBranch: KSFWVariables__bohso12__bc, 0.01037
TBranch: KSFWVariables__bohso14__bc, 0.02871
TBranch: KSFWVariables__bohso20__bc, 0.5172
TBranch: KSFWVariables__bohso22__bc, -0.09269
TBranch: KSFWVariables__bohso24__bc, -0.09905
TBranch: KSFWVariables__bohoo0__bc, 0.2952
TBranch: KSFWVariables__bohoo1__bc, -0.01694
TBranch: KSFWVariables__bohoo2__bc, 0.03508
TBranch: KSFWVariables__bohoo3__bc, -0.00104
TBranch: KSFWVariables__bohoo4__bc, -0.002237
TBranch: CleoConeCS__bo1__bc, 6.184
TBranch: CleoConeCS__bo2__bc, 0.5206
TBranch: CleoConeCS__bo3__bc, 0.5912
TBranch: CleoConeCS__bo4__bc, 0.7903
TBranch: CleoConeCS__bo5__bc, 0
TBranch: CleoConeCS__bo6__bc, 0.1323
TBranch: CleoConeCS__bo7__bc, 0.376
TBranch: CleoConeCS__bo8__bc, 0
TBranch: CleoConeCS__bo9__bc, 2.955
